.github/release/public_release.py

```
@@ -0,0 +1,717 @@
+    return parsed.astimezone(timezone.utc)
+
+
+def _default_verifier(manifest_path: Path, version: str, tag: str, commit: str):
+    root = Path(__file__).resolve().parents[2]
+    command = [
+        "cargo", "run", "--locked", "--quiet", "--manifest-path",
+        str(root / ".github/release/verifier/Cargo.toml"), "--",
+        str(manifest_path), str(root / "crates/tapid-signatures/data/release-keyring.json"),
+        version, tag, commit,
+    ]
+    result = subprocess.run(command, cwd=root, text=True, capture_output=True)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)